src/EnrollTotpController.js

```diff
@@ -111,9 +110,15 @@ const EnrollTotpControllerEnrollTotpController = FormController.extend({
           View: EnrollTotpControllerAppDownloadInstructionsView,
           showWhen: showWhenDeviceTypeSelected,
         }),
-        FormType.Toolbar({
-          noCancelButton: true,
-          save: loc('oform.next', 'login'),
+        FormType.Button({
+          title: loc('oform.next', 'login'),
+          attributes: {
+            'data-type': 'save'
+          },
+          className: 'button button-primary default-custom-button',
+          click: function() {
+            this.model.save();
```

Comment:
this seems to me is doing the same as the `autoSave` (calling model.save when the form is saved). 
Can you expand on how this solves the issue?

Reply:
it does same except for `model.save` will be called only on click of this button. If we have` autoSave = true` and toolbar `submit` button, model.save is called behind the scene for every `enter` key event on every `input` element.

Reply:
https://user-images.githubusercontent.com/94395085/162847455-7ca7a891-1d88-4e0e-83ff-eb3c1e20cd7d.mp4

I have explained the issue here.

Reply:
I think we should make the selection of either iOS or Android mandatory, so the form cannot be submitted without the user selecting an option. This can be done in addition to what you have, but it might also solve the issue on its own.

Reply:
Talked offline about this. It seems that we have extra processing for Radio events (keyup specifically) and that is triggering a form save. So the changes in this PR will avoid that. Let's just add a comment in the code to remember why we had to do this instead of using autoSave